### ISSUE TITLE
Idea: Add outer deadzone for Thumbstick Settings

### ISSUE THREAD

**Issue by BOLL7708** [ValveSoftware/openvr#1859]
## Problem
1. The thumbsticks on the Index controllers deteriorate over time, which in my case is expressed by the received input not moving to the outer edge when moving the stick around in a maxed out circle. 
2. This means in games like the Half-Life 2: VR Mod where the default input is jump on right stick up and crouch on down, such input might not work consistently, causing frustration and mishaps. 
3. Which is what happened to me yesterday and thus I'm writing this post.

## Suggestion
We already have the dialog for Thumbstick Settings, I propose adding a secondary deadzone on the outer rim of the input space, to allow for Index thumbsticks to be useful for a bit longer than what is currently the case.

**Comment by BOLL7708** [comment]
Revisiting this, I was playing Humanity, and getting frustrated that I could not have my people push the blocks. Turns out that needs the stick to be fully actuated, and I don't get that either for straight up or straight down. Took me a bit to realize that as normal movement was fine 😅

**Comment by mdovgialo** [comment]
You can do it in the controller bindings, although per game basis. Click the cog below the thumbstick actions and you can set min/max deadzones and non-linearity.

It would be nice to be able to set that globally for the controllers though.

**Comment by BOLL7708** [comment]
Yeah this would be in the global SteamVR settings, not per game, but nice to know it exists in there. Cheers!
